package.json

```diff
@@ -16,7 +19,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "server": "json-server --watch db.json --port 5000",
+    "dev": "concurrently \"yarn server\" \"yarn start\""
```

Comment:
now, instead of running `yarn start`, we'll be running `yarn dev` that internally will start up our "backend" and start our react app concurrently